idx/memory/memory.go

```diff
@@ -1110,6 +1110,62 @@ func (m *UnpartitionedMemoryIdx) FindByTag(orgId uint32, query tagquery.Query) [
 	return results
 }
 
+func (m *UnpartitionedMemoryIdx) FindTerms(orgID uint32, tags []string, query tagquery.Query) (uint32, map[string]map[string]uint32) {
+	if !TagSupport {
+		log.Warn("memory-idx: received tag query, but tag support is disabled")
+		return 0, nil
+	}
+
+	queryCtx := NewTagQueryContext(query)
+
+	needsName := false
+	terms := make(map[string]map[string]uint32)
+	for _, tag := range tags {
+		if tag == "name" {
+			needsName = true
+		}
+		terms[tag] = make(map[string]uint32)
+	}
+
+	m.RLock()
+	defer m.RUnlock()
+
+	// TODO - terms for meta tags?
```

Comment:
let's make up our mind. is it something to do (when?), or is this something we're deciding not to support for now?

Reply:
Yes, this TODO was meant to spark discussion (forgot to include it when entering the PR). We don't have a need for meta tags in this spot, but maybe it should be supported anyway?

Reply:
How much work would it be to enrich here? cc @replay

Reply:
would be easy and relatively fast, basically just this needs to be done: https://github.com/grafana/metrictank/blob/1735500f2604288c8e610d5f7c1f26d99f69479f/idx/memory/memory.go#L1095

Reply:
I think this is the only remaining point. I'm not entirely sure when we would want to include meta tags for this endpoint, since, as @robert-milan mentions, meta tags are really just tied to other tags (albeit in a non-trivial way). Does anyone have any strong preference?